charts/caasperli/README.md

```diff
@@ -0,0 +1,45 @@
+caasperli
+=========
+Deploy Caasperli to a Kubernetes Cluster
+
+Current chart version is `0.6.1`
+
+Source code can be found [here](https://github.com/adfinis-sygroup/potz-holzoepfel-und-zipfelchape)
+
+
+
+## Chart Values
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| affinity | object | `{}` |  |
+| fullnameOverride | string | `""` |  |
+| image.pullPolicy | string | `"Always"` | When to pull the container image |
+| image.repository | string | `"adfinissygroup/potz-holzoepfel-und-zipfelchape"` | Container image to deploy |
```

Comment:
I'm missing a image.tag here. We should at least make it possible to override the value from `Chart.AppVersion`.

Reply:
True...

https://github.com/adfinis-sygroup/helm-charts/blob/58c7d791cd99c82a1f006be33e98093cd02c6f26/charts/caasperli/templates/deployment.yaml#L28

This is also something I'd rather want adressed in what `helm create` generates.

Reply:
OH I should resync with Helm 3.2.0 which was released since and has this patch: https://github.com/helm/helm/commit/d80e6ae0d82ff3c39cfb596b13d0ac731eadc8e1

Reply:
OH I should resync with Helm 3.2.0 which was released since and has this patch: https://github.com/helm/helm/commit/d80e6ae0d82ff3c39cfb596b13d0ac731eadc8e1